### ISSUE TITLE
Annotating Images

### ISSUE THREAD

**Issue by uskudarli** [bogaziciswe/b.w.a.t#113]
I am trying to annotate an image, I can not figure out how to select the image part. 

The manual says: Select the part of the image (The Man Who Would Teach Machines to Think).

But, when I try to select an image, I only see the "text cursor".

**Comment by uskudarli** [comment]
Can someone please tell me how image annotation is supposed to work? I am not able to.

**Comment by ozkuran** [comment]
Can you see the "Click and Drag To Annotate" text on the top left of the image when cursor get over to the image?

**Comment by uskudarli** [comment]
No, I do not.

Nothing happens when the cursor is over the image. Hovering or clicking makes nothing.

**Comment by ozkuran** [comment]
Just to be sure if it is a network or other configuration issue, could you please check if can you annotate image on [http://www.cchaven.com/](http://www.cchaven.com/) page. It is and old page without any server side scripting.

**Comment by uskudarli** [comment]
Does not work there either. 

Let me disable all my other extensions just to see.

**Comment by uskudarli** [comment]
There is a notice on Chrome when I install the plugin: 

There were warnings when trying to install this extension:
Unrecognized manifest key 'applications'.

I this normal?

**Comment by uskudarli** [comment]
The extension will not load on Firefox as it was supposed to.

But, we have been testing on Chrome since yesterday anyway. 

I am on Chrome on Ubuntu/Linux

**Comment by uskudarli** [comment]
Are you able to annotate http://mitan.in/bcp/raika/ images?

**Comment by ozkuran** [comment]
"Unrecognized manifest key 'applications'." Is about version difference in the browser we need to create it dynamically I am opening a new issue for it.

**Comment by ozkuran** [comment]
Yes i can annotate. Here You may find an example image annotation below.


![https://raw.githubusercontent.com/bogaziciswe/b.w.a.t/master/reportsAndLogs/Reports/camel.gif](https://raw.githubusercontent.com/bogaziciswe/b.w.a.t/master/reportsAndLogs/Reports/camel.gif)

**Comment by uskudarli** [comment]
I am mystified.

I am on Google Chrome  Version 55.0.2883.87 (64-bit)  on Ubuntu 14.04

What is your setup?

Can you show the bit where "Click and drag" instructions near the image are shown too?

**Comment by ozkuran** [comment]
My setup is Google Chrome Version 55.0.2883.87 m (64-bit) on Windows 10 x64. I think only difference is Linux. We need test thoroughly on Linux setup as we tested the application on OSX and Windows boxes. 

The "Click and drag" message shown in top left of the image that you want to annotate when mouse cursor  get top of the image.

![https://raw.githubusercontent.com/bogaziciswe/b.w.a.t/master/reportsAndLogs/Reports/camel2.png](https://raw.githubusercontent.com/bogaziciswe/b.w.a.t/master/reportsAndLogs/Reports/camel2.png)

**Comment by uskudarli** [comment]
I don't see that. Also, question, is that informative, or do yo uneed to click on it to activate selection process?

**Comment by uskudarli** [comment]
Also, on that page, I see the rectangle related to your annotation I presume. But the body of the annotation does not show up on the side panel. I created a new annotation. That one shows up. Initially there was nothing there. I expect to see the annotation you made.

**Comment by ozkuran** [comment]
I think that is another bug we need to fix this to show image annotations on the side panel. 
It is just informative. The annotation process of the image is similar to the text annotation. When the plugin activates all images on the page become available to annotate, and all you need is just click and draw the rectangle on the part of the image you wish to annotate. When you leave the mouse button the annotation box shows up under the image and waits you to enter annotation body.

**Comment by gokhan-ozgozen** [comment]
![imann](https://cloud.githubusercontent.com/assets/9518468/21751221/618f4bc0-d5cc-11e6-8fde-94a922c3bf59.png)
I managed to annotate Images on Ubuntu/Chrome. The important thing is to wait until all background scripting and requests are completed to get it work. For this particular instance, I waited hurriyet.com's all background tasks ( it created 361 requests and I waited around 15 seconds ) BEFORE making any image annotation.
![imannw](https://cloud.githubusercontent.com/assets/9518468/21751243/af847de6-d5cc-11e6-99e4-75a69ca3594f.png)

By the way, I use Ubuntu 14.04 and Chrome 53.0.2785.116 (64-bit).
![osv](https://cloud.githubusercontent.com/assets/9518468/21751270/4e64e5ea-d5cd-11e6-9789-ee19a2b5c921.png)
![cv](https://cloud.githubusercontent.com/assets/9518468/21751272/516e05d2-d5cd-11e6-93fc-33776f3cff9a.png)

**Comment by uskudarli** [comment]
so, tell me the steps please. I will try once more.

**Comment by uskudarli** [comment]
I tried again, my version is 55, but I don't think that should explain the problem.

When I click on it, and observe developer tools it doesn't seem to respond like yours.

**Comment by uskudarli** [comment]
ok. i guess we give up. i will see if i can find another computer tomorrow to test.

**Comment by oakbas** [comment]
On the side bar, image annotations are also added, but original source image is shown, not cropped image, but cropped image information x-y-width and height is written.

**Comment by oakbas** [comment]
<img width="1211" alt="screen shot 2017-01-09 at 15 07 35" src="https://cloud.githubusercontent.com/assets/15357336/21766075/d00f39f2-d685-11e6-8bc1-cba2fc954397.png">
